tests/foreman/cli/test_organization.py

```diff
@@ -1162,6 +1164,176 @@ def test_positive_remove_capsule_by_name(self):
         org = Org.info({'name': org['name']})
         self.assertNotIn(proxy['name'], org['smart-proxies'])
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_add_location_by_id(self):
+        """Add a location to organization by its id
+
+        @id: 83848f18-2cca-457c-af57-e6249386c81c
+
+        @Assert: Location is added to the org
+
+        @CaseLevel: Integration
+        """
+        org = make_org()
+        loc = make_location()
+        Org.add_location({
+            'location-id': loc['id'],
+            'name': org['name'],
+        })
+        result = Org.info({'id': org['id']})
+        self.assertEqual(len(result['locations']), 1)
+        self.assertIn(loc['name'], result['locations'])
+
+    @run_only_on('sat')
+    @tier2
+    def test_positive_add_location_by_name(self):
+        """Add a location to organization by its name
+
+        @id: f39522e8-5280-429e-b954-79153c2c73c2
+
+        @Assert: Location is added to the org
+
+        @CaseLevel: Integration
+        """
+        org = make_org()
+        loc = make_location()
+        Org.add_location({
+            'location': loc['name'],
+            'name': org['name'],
+        })
+        result = Org.info({'id': org['id']})
+        self.assertEqual(len(result['locations']), 1)
+        self.assertIn(loc['name'], result['locations'])
+
+    @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1395229)
```

Comment:
Today, I created a bug that seems much more relevant:
https://bugzilla.redhat.com/show_bug.cgi?id=1408940